flipt-client-go/README.md

```diff
@@ -30,7 +30,7 @@ func main() {
   // flipt.WithNamespace(string): configures which namespace you will be making evaluations on
   // flipt.WithURL(string): configures which upstream Flipt data should be fetched from
   // flipt.WithUpdateInterval(int): configures how often data should be fetched from the upstream
-  // flipt.WithAuthToken(string): configures an auth token if your upstream Flipt instance requires it
+  // flipt.WithAuthentication(any): configures an authentication method if your upstream Flipt instance requires it
```

Comment:
just FYI this is gonna be a breaking change across all clients, which I think is ok since we are in the `0.0.x` range, but this will likely be the `0.1.0` release

Reply:
@markphelps Yeah noted!